### ISSUE TITLE
en.yml changes in Angular JSS app giving error : Duplicate function implementation

### ISSUE THREAD

**Issue by alexksharma** [Sitecore/jss#326]
I just created a new Angular App and its running fine with "JSS Build". But when you change any of the en.yml files, those changes are not reflected in browser and I am getting below error on command prompt
path or pattern './sitecore/pipelines/**/*.patch.js' did not match any files.
sitecore/definitions/component-content.sitecore.ts:32:10 - error TS2393: Duplicate function implementation.
32 function convertToItems(data, basePath, rootItemName, language) {
            ~~~~~~~~~~~~~~
sitecore/definitions/component-content.sitecore.ts:73:10 - error TS2393: Duplicate function implementation.
73 function convertPhsyicalPathToItemRelativePath(physicalPath, basePath) {

Do anybody know the reason and possible fix?

**Comment by adambezemek** [comment]
@alexksharma - Did you ever figure this out? I am battling the same issue at the moment.

**Comment by sc-illiakovalenko** [comment]
@alexksharma Fixed in #440 
Will be released in JSS 15
